events/errors.js

```diff
@@ -0,0 +1,19 @@
+class InvalidStatus extends Error {
+  constructor() {
+    super();
+    this.status = 400;
+    this.message = 'Invalid status';
+  }
+}
+class StatusNotHandled extends Error {
```

Comment:
Maybe this should be amended to `TypeNotHandled` to prevent confusion between this and the InvalidStatus check?